Update eslint-plugin-react to v6

martinnorum opened the pull request:
allow-in-func is now default behavior for react/no-did-mount-set-state:
(https://github.com/yannickcr/eslint-plugin-react/commit/376dc53656f78059fb405d9e30c2f402717f6882)

martinnorum committed:
Remove allow-in-func from react/no-did-mount-set-state

allow-in-func is now default behavior for react/no-did-mount-set-state:
(https://github.com/yannickcr/eslint-plugin-react/commit/376dc53656f78059fb405d9e30c2f402717f6882)

keithamus commented:
Still using 5.0.1: https://github.com/keithamus/eslint-config-strict-react/blob/master/package.json#L71. Fancy making a PR to update version and add new rules?

martinnorum commented:
Ah, missed that :-)

Still new to this whole open sorce thingy….

Don’t have the time to fix it right now as I’m just trying to just build my
code with updated deps and then go to lunch. But later :-)
- M

man. 8. aug. 2016 kl. 12.04 skrev Keith Cirkel notifications@github.com:

> Still using 5.0.1:
> https://github.com/keithamus/eslint-config-strict-react/blob/master/package.json#L71.
> Fancy making a PR to update version and add new rules?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/keithamus/eslint-config-strict-react/pull/18#issuecomment-238193933,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AAq05uUUrEwFF3-8njFvaHD39rNsG4cgks5qdv8pgaJpZM4Je4mI
> .

keithamus commented:
np @martinnorum. I'll stick it on my todo list and get around to it eventually, if you don't beat me to it 😄

martinnorum committed:
chore(package): update eslint to version 3.2.2

martinnorum committed:
chore(package): update eslint-plugin-react to version 6.0.0

martinnorum committed:
fix(rules): Remove allow-in-func

allow-in-func is now default and
is replaced by disallow-in-func

https://github.com/yannickcr/eslint-plugin-react/commit/376dc53656f78059fb405d9e30c2f402717f6882

martinnorum committed:
fix(rules): Replace react/wrap-multilines

react/wrap-multilines is deprecated and
replaced with react/jsx-wrap-multilines

martinnorum committed:
Merge branch 'patch-1' of github.com:martinnorum/eslint-config-strict-react into patch-1

martinnorum committed:
fix(package): downgrade eslint dep to 2.5.0
